I,guess,this,is,an,old,thread,",",but,i,was,having,the,same,issue,so,thought,I'd,share,my,solution,in,case,someone,else,runs,into,this,same,page,.,basically,the,error,comes,in,when,trying,to,run,mencoder,from,outside,your,source,folder,location,.,so,to,get,this,to,work,",",just,CD,into,the,folder,with,your,source,images,and,run,the,command,from,there,.,OR,you,can,modify,your,python,script,and,add,a,line,changing,the,current,path,to,your,source,image,path,just,before,running,mencoder,.,Something,like,:,os.chdir(sourcefolder),#,where,sourcefolder,is,the,location,of,your,source,images,Cheers,!,-,Kevin
